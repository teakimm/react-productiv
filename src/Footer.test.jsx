import { describe, it, expect, test } from "vitest";

import React from "react";
import { fireEvent, render } from "@testing-library/react";
import Footer from "./Footer";

it("renders without crashing", function () {
    render(<Footer />);
});

it("matches snapshot", function () {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
});

it("renders correct elements", function () {
    const result = render(<Footer />);


    expect(
        result.queryByText("Prødutïv is copyright ©2020 by Flüffy Data Enterprises, Inc.")
    ).toBeInTheDocument();
});