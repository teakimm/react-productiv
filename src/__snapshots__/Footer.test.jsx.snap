// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<div>
  <footer
    class="App-footer mt-5 text-muted"
  >
    <small>
      Prødutïv is copyright ©2020 by Flüffy Data Enterprises, Inc.
    </small>
  </footer>
</div>
`;
